src/Microsoft.CodeQuality.Analyzers/Core/Maintainability/UseNameofInPlaceOfString.cs

```diff
@@ -61,11 +61,6 @@ private void AnalyzeArgument(OperationAnalysisContext context)
                 return;
             }
 
-            if (argument.Parameter == null)
```

Comment:
The hover popup says that `Parameter` can't be null so I am removing this un-needed/unreachable early exit.

Reply:
The hover popups are not a guarantee. In this case, the definition of `IArgumentOperation` is null oblivious.

Reply:
Ah too bad! I thought that was something based on nullability declaration,

Reply:
I believe `IArgumentOperation.Parameter` cannot be null. @333fred to confirm.